flux-manifests/mimir.yaml

```diff
@@ -0,0 +1,13 @@
+api_version: generators.giantswarm.io/v1
+app_catalog: control-plane-catalog
+app_destination_namespace: mimir
+app_name: mimir
+app_version: 0.6.3
+kind: Konfigure
+metadata:
+  annotations:
+    config.kubernetes.io/function: |-
+      exec:
+        path: /plugins/konfigure
```

Comment:
What is this magic doing ?

Reply:
It's running the konfigure cli tool to template the config I think? That's what's been used for years not